browser/brave_rewards/BUILD.gn

```diff
@@ -3,8 +3,70 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import("//brave/components/greaselion/browser/buildflags/buildflags.gni")
+import("//extensions/buildflags/buildflags.gni")
 import("//testing/test.gni")
 
+source_set("brave_rewards") {
+  sources = [
+    "//brave/browser/brave_rewards/rewards_service_factory.h",
+    "//brave/browser/brave_rewards/rewards_tab_helper.h",
```

Comment:
nit: consider fixing these paths to be local in all new targets you introduce in this PR:
```
    "rewards_service_factory.h",
    "rewards_tab_helper.h",
```